app/models/group.rb

```diff
@@ -604,4 +604,12 @@ def limit_inheritance
   def self.with_one_coordinator
     published.select{ |g| g.admins.count == 1 }
   end
+
+  def default_cover_photo
+    if self.default_group_cover
+      /^.*(?=\?)/.match(self.default_group_cover.cover_photo.url).to_s
```

Comment:
Can we just migrate all groups without custom covers to have a DefaultGroupCover? I think that would be fine